recipe/meta.yaml

```diff
@@ -12,26 +12,32 @@ source:
     - byte-order-osx.patch
 
 build:
-  number: 2
+  number: 0
   # Not supported on win
   skip: True   # [win]
 
 requirements:
   build:
-    - gnuconfig  # [unix]
+    - gnuconfig
     - {{ compiler('c') }}
     - make
     - bison
+    - gawk
     - flex
     - pkg-config
+    - patch     # [not win]
+    - m2-patch  # [win]
```

Comment:
`m2-patch` is redundant if do not build for `win`

Reply:
true, but it silence linter